src/utilities/stir_config_dir.cxx

```diff
@@ -0,0 +1,54 @@
+/*!
+  
+  Copyright (C) 2021, National Physical Laboratory
+  This file is part of STIR.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  See STIR/LICENSE.txt for details
+  \file
+  \ingroup utilities
+  \brief Print configuration directory
```

Comment:
as this can print version now, this doc needs to be adapted. Give usage here I guess.

Also, we should rename the file to `stir_config.cxx`